azure/scope/managedcontrolplane.go

```diff
@@ -642,14 +664,24 @@ func (s *ManagedControlPlaneScope) MakeEmptyKubeConfigSecret() corev1.Secret {
 	}
 }
 
-// GetKubeConfigData returns a []byte that contains kubeconfig.
-func (s *ManagedControlPlaneScope) GetKubeConfigData() []byte {
-	return s.kubeConfigData
+// GetAdminKubeconfigData returns admin kubeconfig.
+func (s *ManagedControlPlaneScope) GetAdminKubeconfigData() []byte {
+	return s.adminKubeConfigData
+}
+
+// SetAdminKubeconfigData sets adminKubeconfig data.
```

Comment:
nit for consistency with the comment above
```suggestion
// SetAdminKubeconfigData sets admin kubeconfig data.
```